zerver/openapi/zulip.yaml

```diff
@@ -3966,6 +3976,11 @@ components:
           type: boolean
           description: |
             A boolean specifying whether the user is an organization administrator.
+        is_owner:
+          type: boolean
+          description: |
+            A boolean specifying whether the user is an organization owner.
+            If true, is_admin will also be true.
```

Comment:
Any changes to `zulip.yaml` as part of an API changed need a `**Changes**:` entry.  I added this one:

```
+
+            **Changes**: New in Zulip 2.2 (feature level 8).
```

Reply:
Yeah, will be taken care from next time.